makecrx.sh

```diff
@@ -131,4 +131,4 @@ if [ -n "$BRANCH" ]; then
   cd ..
   cp $SUBDIR/$crx pkg
   rm -rf $SUBDIR
-fi
+fi
```

Comment:
It looks like your text editor removes newlines at the end of files, which generates spurious diffs. Could you either configure it not to do so, or manually add an extra newline so you don't generate these diffs?